openstack/imageservice/v2/members/results.go

```diff
@@ -0,0 +1,83 @@
+package members
+
+import (
+	"github.com/mitchellh/mapstructure"
+	"github.com/rackspace/gophercloud"
+)
+
+// ImageMember model
+type ImageMember struct {
+	CreatedAt string `mapstructure:"created_at"`
```

Comment:
This and `UpdatedAt` should be `time.Time`s similar to here: https://github.com/rackspace/gophercloud/blob/master/openstack/compute/v2/servers/urls.go
(and see the `Extract` method there for the implementation"